apps/change-log.md

```diff
@@ -10,6 +10,24 @@ In order to keep track of these changes we recommend you to follow [@epagesdevs]
 
 <hr>
 
+## 2016-04-11
+
+### Changes with software release 6.17.43
+
+#### <i class="fa fa-plus"></i> New
+
+* [`POST` products](https://developer.epages.com/apps/api-reference/post-shops-shopid-products.html)
+* [`GET` watched-products](https://developer.epages.com/apps/api-reference/get-shops-shopid-watched-products.html)
+* [`GET` customers](https://developer.epages.com/apps/api-reference/get-shops-shopid-customers-customerid.html)
+* [`POST` carts/{cartId}/order](https://developer.epages.com/apps/api-reference/post-shops-shopid-carts-cartid-order.html)
+
+#### <i class="fa fa-pencil"></i> Update
+
+* Added `taxClass` attribute to [`GET` products/{productId}](https://developer.epages.com/apps/api-reference/get-shops-shopid-products-productid.html)
+* Added `stocklevel` attribute for products_write authorissation to [`GET` products/{productId}](https://developer.epages.com/apps/api-reference/get-shops-shopid-products-productid.html) and [`PATCH` products/{productId}](https://developer.epages.com/apps/api-reference/patch-shops-shopid-products-productid.html)
```

Comment:
better write it like:
added stocklevel attribute to GET bla for requests with *products_write* scope